src/parts/_meter_progress.css

```diff
@@ -0,0 +1,58 @@
+#meter-wrapper {
+  width: 640px;
+}
+
+h1 {
+  margin-top: 0;
+}
+
+.styled progress {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  width: 60%;
+  height: 25px;
+}
+
+.styled progress::-webkit-progress-bar {
+  background: #eee;

```

Comment:
We would want to use the variables in the light/dark theme files instead of creating new colors here. Something like `--backround-alt` might be appropriate here!